src/adk/api/qne_client.py

```diff
@@ -454,6 +456,11 @@ def retrieve_backend(self, backend_url: str) -> BackendType:
         response = self._action('retrieveBackend', id=backend_id)
         return cast(BackendType, response)
 
+    def list_backends_networks(self) -> List[Any]:
```

Comment:
maybe rename to list_backend_network_mappings?

Reply:
I deleted it. It was a placeholder for the endpoint we will not add to the api-router